docs/data/charts/sparkline/CustomDomainYAxis.js

```diff
@@ -11,21 +11,51 @@ const settings = {
   showHighlight: true,
 };
 
-const values = [60, -25, 66, 68, 87, 82, 83, 100, 92, 75, 76, 50, 91];
+// Extend a value to match a multiple of the step.
+function extend(value, step) {
+  if (value > 0) {
+    // If >0 go to the next step
+    return step * Math.ceil(value / step);
+  }
+  // If <0 go to the previous step
+  return step * Math.floor(value / step);
+}
+
+const values = [60, -15, 66, 68, 87, 82, 83, 85, 92, 75, 76, 50, 91];
```

Comment:
I replaced the 100 by 92 such that nice and strict don't leads to the same result